ldclient/client.py

```diff
@@ -27,79 +25,10 @@
 from cachecontrol import CacheControl
 from threading import Lock
 
-GET_LATEST_FEATURES_PATH = '/sdk/latest-flags'
```

Comment:
In order to make this change backwards compatible, I think you could alias the `Config` class in this package